### DISCUSSION TITLE
Coingecko price not fetching

### DISCUSSION THREAD

**Discussion by Osman-Mahmood** [blockscout/blockscout#8774] [Q&A]
Hi @vbaranov 
Sir we got a problem in fetching coingecko price of our coin
DISABLE_EXCHANGE_RATES=false
and i am adding 
EXCHANGE_RATES_MARKET_CAP_SOURCE=coin_gecko
EXCHANGE_RATES_PRICE_SOURCE=coin_gecko
EXCHANGE_RATES_COINGECKO_COIN_ID=coin name
EXCHANGE_RATES_COINGECKO_API_KEY=created new api key in coin gecko and add it here
EXCHANGE_RATES_COINGECKO_PLATFORM_ID=added coin name
backend                         | 2023-11-02T10:17:00.302 application=explorer [warning] Failed to get exchange rates with reason ''.
please guide us it's possible solution
thanks,

**Comment by sl1depengwyn** [comment]
`EXCHANGE_RATES_COINGECKO_COIN_ID` and `EXCHANGE_RATES_COINGECKO_PLATFORM_ID` are important values to resolve this problem, could you please share this?

**Comment by Osman-Mahmood** [comment]
hi here is my env
EXCHANGE_RATES_COINGECKO_API_KEY=CG-***************
EXCHANGE_RATES_MARKET_CAP_SOURCE=coin_gecko
EXCHANGE_RATES_PRICE_SOURCE=coin_gecko
EXCHANGE_RATES_COINGECKO_PLATFORM_ID=virtual-coin
EXCHANGE_RATES_COINGECKO_COIN_ID=virtual-coin
